src/main/resources/columns.properties

```diff
@@ -12,7 +12,7 @@ hotel.name = name
 hotel.city = city
 hotel.country = country
 hotel.numberOfStars = number_of_stars
-tourOffer.tableName = tour_offer
+tourOffer.tableName = tourOffer
```

Comment:
hmm, what was wrong with the previous version of the name? 'Cause this one doesn't match with name convention

Reply:
This and other names will be fixed in another PR about fixing names, for now we need to fix the logic

Reply:
@imbirevkotlya Do you mean that i need to make method in DAO layer more simpler?